log.js

```diff
@@ -72,5 +72,6 @@ if (appInsights) {
     return flattenMeta(meta)
   })
 }
-
-module.exports = logger
+/** @type {winston.LoggerInstance & {audit?: Function }} */
```

Comment:
Winston adds additional functions for each logger level defined which can't be covered by the ts definition. 
This defines an [intersection type](https://www.typescriptlang.org/docs/handbook/advanced-types.html#intersection-types) which is both a winston logger and one of these anonymous types which may have an audit function.

This allows us to call `logger.audit(message)` in `audit.js`